### ISSUE TITLE
[Bug]: release broken

### ISSUE THREAD

**Issue by glaszig** [orchidjs/tom-select#645]
### Bug description

version 2.2.3 package produces the following error:

```
SassC::SyntaxError - Error: File to import not found or unreadable: ../plugins/drag_drop/plugin.
        on line 74:1 of node_modules/tom-select/dist/scss/tom-select.scss
>> @import "../plugins/drag_drop/plugin";
   ^:
``` 

as i'm doing this:

```
@import "tom-select/dist/scss/tom-select";
```

the problem is that there is no such file `../plugins/drag_drop/plugin.scss`.
it lives at `tom-select/dist/scss/plugins/drag_drop.scss`.

as development depends on the former location and the release version puts files in another place i'm not sure how to fix this in a pr.

### Expected behavior

no error.

### Steps to reproduce

```
@import "tom-select/dist/scss/tom-select";
```

### Additional context

version 2.2.3

**Comment by oyejorge** [comment]
This is a duplicate, you should be able to find the original by looking in
the recently closed bugs

**Comment by glaszig** [comment]
still broken though. see #642.

<img width="838" alt="screenshot" src="https://github.com/orchidjs/tom-select/assets/201135/be99b1fd-c560-4bb0-98d7-f758ead1b419">

**Comment by glaszig** [comment]
the import statements in the dist scss are still wrong 👀

**Comment by glaszig** [comment]
when i say "dist scss" i mean the content of the package as it comes as 2.2.3 from npm right now. see screenshot.

**Comment by gumeson-zipnosis** [comment]
I'm also getting this with version `2.2.2`. Only way I can fix is by pinning to version `2.2.1`.

**Comment by glaszig** [comment]
yeah it’s right there in the screenshot of the package source. don’t understand how it gets just disregarded.
